modules/tensor_mechanics/include/utils/RankTwoScalarTools.h

```diff
@@ -12,6 +12,16 @@ class RankTwoTensor;
 namespace RankTwoScalarTools
 {
 /*
+ * Return the quantity_type MooseEnum
+ */
+MooseEnum scalarOptions();
+
+/*
+ * Return components and equivalent quantity of rank two tensor based on the user specified quantity_type
+ */
+Real getQuantity(const RankTwoTensor & tensor, const MooseEnum scalar_type, const Point & point1, const Point & point2, const Point * curr_point, Point & direction);
+
```

Comment:
Please add in the @ params to describe what the points ```point1``` and ```point2``` are in the Doxygen -- we have seen questions about these parameters in the Moose Users list.

Will getQuantity need to be called every time to access the scalar functions in this namespace?  I'd like to avoid a situation where we need to pass in two unused points (point1 and point2) to calculate von Mises stress or the first invariant as is done in Solid Mechanics.